package.json

```diff
@@ -106,7 +106,7 @@
         "angular2-text-mask": "9.0.0",
         "classlist.js": "^1.1.20150312",
         "core-js": "^2.6.9",
-        "dompurify": "^2.2.3",
+        "dompurify": "^2.3.5",
```

Comment:
Some versions are incompatible with SSR, it's better to fix it, I think:
https://github.com/Tinkoff/ng-dompurify/issues/65